### ISSUE TITLE
Make the numerical entry/slider/spinner controls not the worst

### ISSUE THREAD

**Issue by chaomodus** [glimpse-editor/Glimpse#266]
### Is your feature request related to a problem? Please describe
For some time every numerical entry that represents a range (for example setting the opacity of a layer, but almost every numerical range entry is one of these controls), is somewhat difficult to use without special knowledge. 

### Describe the solution you'd like
I'm not completely certain of the solution but other UIs don't seem to have this issue so it must be something about the particular controls or how the UX is handled. In this case, it's completely confusing that single clicking slowly scrolls the value (much more slowly than should be necessary, I recognize that this is fine tuning but it is the default interaction) - double clicking 'sets' the value *and* activates the entry so a mis-button can clear the value completely, and the mouse is modally locked to fast scrolling / setting the value. It's just a mess from a usability standpoint. 

In summary: The solution would be to change this control to a less confusing / modal version.

**Comment by chaomodus** [comment]
One thing that jumps out at me as a possible immediate fix is to make the 'set' value (*not* the entry) the default single click interaction, so you're basically setting how 'full' the control is by just clicking it, and perhaps middle/right click slowly adjusts it, and double clicking allows you to enter a value. I believe this is similar to the original incarnation of these same controls.

**Comment by TrechNex** [comment]
Can you provide some screenshots in this issue entry @chaomodus? It will help me to size the task and determine when we can do it :)
